community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/internal/CypherAdapterStream.java

```diff
@@ -72,7 +80,7 @@ public boolean visit( Result.ResultRow row ) throws Exception
         } );
 
         QueryExecutionType qt = delegate.getQueryExecutionType();
-        visitor.addMetadata( "type", queryTypeCode( qt.queryType() ) );
+        visitor.addMetadata( "type", queryType );
```

Comment:
I think it is fair to remove `type` from `PULL_ALL` as there is no need to stream to client the same info twice